### ISSUE TITLE
Deconvolutional layer in model_01_deconv.py

### ISSUE THREAD

**Issue by OneManArmy93** [yukitsuji/kitti-py#37]
Can anyone explain what the deconvolutional layer `def deconv3D_to_output(input_layer, input_dim, output_dim, height, width, length, stride, output_shape, activation=tf.nn.relu, padding="SAME", name="")` is for? and does is influence the training/learning process?
Thank you.
